readthedocs/doc_builder/python_environments.py

```diff
@@ -381,11 +381,7 @@ def install_core_requirements(self):
                     negative='sphinx<2',
                 ),
                 'sphinx-rtd-theme<0.5',
-                self.project.get_feature_value(
-                    Feature.USE_SPHINX_RTD_EXT_LATEST,
-                    positive='readthedocs-sphinx-ext',
-                    negative='readthedocs-sphinx-ext<1.1',
```

Comment:
Instead of removing the feature flag, we may want to update the version number here and remove the sampled projects once it's released.

Otherwise, when we publish a new version of the extension, all the projects will install automatically the just released one.

Reply:
ping @ericholscher

Reply:
That's probably a better idea 👍